client/post-editor/controller.js

```diff
@@ -262,6 +262,11 @@ export default {
 			return next();
 		}
 
+		// Bypass the selected editor check if the URL contains a force=true param
+		if ( get( context.query, 'force', false ) ) {
+			return next();
```

Comment:
Oh wait, we probably want to move this down into the store subscribe callback, so siteId is available.

Reply:
I'm afraid I don't see your point. Why do we need `siteId`? Is not only needed for checking the editor of the current site? We don't need to check the editor when `force` is present, as we'll redirect always to the classic editor.

Reply:
Presumably the editor breaks in some way without the siteId, or we wouldn't wait for it in the controller.

Reply:
`siteId` is needed in `maybeCalypsoifyGutenberg` and this function waits for it. But when the `force` param is present, `maybeCalypsoifyGutenberg` is not executed, and [we call to the next controller](https://github.com/Automattic/wp-calypso/blob/13344ccd07c052a3fcb1c3b9cf90776173730775/client/post-editor/controller.js#L308).

Reply:
Gotcha 👍. We can always follow up in other PRs if folks notice if we missed any corner cases.